packages/components/src/molecules/Card/Card.tsx

```diff
@@ -0,0 +1,61 @@
+import type { HTMLAttributes } from "react";
+import React, { forwardRef } from 'react'
+
+import { Icon } from '../..'
+
+export interface CardProps extends Omit<HTMLAttributes<HTMLDivElement>, 'role'> {
+    /**
+    * Specifies the text to be loaded into the header.
+    */
+    title?: string
+    /**
+    * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+    */
+    testId?: string
+    /**
+    * Specifies the card max width, if not informed it will be of 300px.
+    */
+    maxWidth?: string
+    /**
+    * Specifies what icon to use following the FastStore Icon sheet.
+    */
+    iconName?: string
+    /**
+    * Specifies the icon's color
+    */
+    iconColor?: string
+    /**
+    * Function that should be executed when the icon is clicked
+    */
+    iconAction?: () => void
+}
+
+const Card = forwardRef<HTMLDivElement, CardProps>(function Card({
+    title,
+    children,
+    maxWidth,
+    testId = 'fs-card',
+    iconName,
+    iconColor,
+    iconAction
+}, ref) {
+    return (
+        <div data-testid={testId} ref={ref} data-fs-card style={{ maxWidth: `${maxWidth}` }}>
+            <div data-fs-card-header style={{ display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
`style={{ display: 'flex', justifyContent: 'space-between' }}>` 

**todo:** Can you please move this inline style to the [/molecules/Card/styles.scss](https://github.com/vtex/faststore/pull/2398/files#diff-ecc6db095738567755bce1001bfebefdfe9daa6031016b2456764b743cde2f63) stylesheet? 

We usually don't use inline css.